src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -6657,20 +6657,20 @@ public class DefaultMemberAttribute { }
             src.WriteAllText(source + mslib);
 
             var outWriter = new StringWriter(CultureInfo.InvariantCulture);
-            int exitCode = new MockCSharpCompiler(null, _baseDirectory, new[] { "/nologo", "/noconfig", "/nostdlib", "/runtimemetadataversion:v4.0.30319", src.ToString() }).Run(outWriter);
+            int exitCode = new MockCSharpCompiler(null, _baseDirectory, new[] { "/nologo", "/noconfig", "/nostdlib", "/runtimemetadataversion:v4.0.30319", "/langversion:7", src.ToString() }).Run(outWriter);
```

Comment:
>"/langversion:7" [](start = 155, length = 16)

Perhaps disabling specific warning is a better long term approach. We do want to run this scenario against the latest language version.